src/containerapp/HISTORY.rst

```diff
@@ -3,7 +3,10 @@
 Release History
 ===============
 upcoming
+
+0.3.41
 ++++++
+* 'az containerapp create': enable support for no Dockerfile cases with --repo
```

Comment:
May I ask if this change only applies to `create` command not including `update` command?

Reply:
```suggestion
* 'az containerapp up/create': enable support for no Dockerfile cases with --repo
```